resources/layout/main.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Asset\AppAsset;
+use Yiisoft\Html\Html;
+
+$assetManager->register([
+    AppAsset::class,
+]);
+
+$this->setCssFiles($assetManager->getCssFiles());
+$this->setJsFiles($assetManager->getJsFiles());
+
+?>
+
+<?php $this->beginPage() ?>
+    <!DOCTYPE html>
+    <?= Html::beginTag('html', ['lang' => $layoutParameters->getLanguage()]) ?>
+
+        <?= $this->render('_head', ['csrf' => $csrf]) ?>
+
+        <?php $this->beginBody() ?>
+
+            <?= Html::beginTag('body') ?>
+
+                <?= Html::beginTag('section', $layoutParameters->getHeroOptions()) ?>
+
+                    <?= Html::beginTag('div', $layoutParameters->getHeroHeadOptions()) ?>
+                        <?= $this->render('_menu') ?>
+                    <?= Html::endTag('div') ?>
+
+                    <?= Html::beginTag('div', $layoutParameters->getHeroBodyOptions()) ?>
+                        <?= Html::beginTag('div', $layoutParameters->getHeroContainerOptions()) ?>
+                            <?= $content ?>
+                        <?= Html::endTag('div') ?>
+                    <?= Html::endTag('div') ?>
+
+                    <?= Html::beginTag('div', $layoutParameters->getHeroFooterOptions()) ?>
+                        <?= $this->render('_footer') ?>
```

Comment:
Do we need to separate it?

Reply:
separate menu, footer and head from the main layout because as a template it is good to have this feature, if you want to create a panel or any system you will have the views in a modular way.